sdks/python/apache_beam/transforms/external.py

```diff
@@ -431,10 +431,15 @@ def __init__(self, urn, payload, expansion_service=None):
     self._payload = (
         payload.payload() if isinstance(payload, PayloadBuilder) else payload)
     self._expansion_service = expansion_service
+    self._output_coder = None
     self._external_namespace = self._fresh_namespace()
     self._inputs = {}  # type: Dict[str, pvalue.PCollection]
     self._outputs = {}  # type: Dict[str, pvalue.PCollection]
 
+  def with_output_coder(self, output_coder):
```

Comment:
PTransforms already have a with_output_type. Let's grab that and (if present) infer the coder rather than adding a new method.

Reply:
Done.